src/commands/admin/appcommands.ts

```diff
@@ -120,17 +137,24 @@ export default class AppCommandsCommand implements BaseCommand {
 
             sendInfoMessage(MessageContext.fromMessage(message), {
                 title: "Application Commands Reloaded",
-                description: "Yay.",
+                description: commandsReloaded.join(", "),
             });
         } else {
-            const commands = await State.client.getCommands();
+            const commandsToModify = isSingleCommand
```

Comment:
This is common between reload and delete

Reply:
Object.keys vs Object.values